tsl/test/isolation/specs/CMakeLists.txt

```diff
@@ -4,22 +4,23 @@ set(TEST_TEMPLATES_MODULE
   reorder_vs_insert_other_chunk.spec.in
 )
 
-set(TEST_TEMPLATES
-  continuous_aggs_insert.spec.in
-  continuous_aggs_multi.spec.in
-)
-
 set(TEST_TEMPLATES_MODULE_DEBUG
   reorder_vs_insert.spec.in
   reorder_vs_select.spec.in
 )
 
+list(APPEND TEST_FILES
+  continuous_aggs_insert.spec
```

Comment:
shouldn't we exclude isolation tests here on PG versions that do not support the new output?

Reply:
Yes, this is what I started to do already, but haven't finished and pushed. I will continue with this path. Thank you!